napari_btrack/track.py

```diff
@@ -1,24 +1,144 @@
+            # so we can read the matrix entries
+            # fmt: off
+            A_cell=[1, 0, 0, 1, 0, 0,
+                    0, 1, 0, 0, 1, 0,
+                    0, 0, 1, 0, 0, 1,
+                    0, 0, 0, 1, 0, 0,
+                    0, 0, 0, 0, 1, 0,
+                    0, 0, 0, 0, 0, 1
+                    ],
+            A_particle=[1, 0, 0, 0, 0, 0,
+                        0, 1, 0, 0, 0, 0,
+                        0, 0, 1, 0, 0, 0,
+                        0, 0, 0, 1, 0, 0,
+                        0, 0, 0, 0, 1, 0,
+                        0, 0, 0, 0, 0, 1],
+            H=[1, 0, 0, 0, 0, 0,
+               0, 1, 0, 0, 0, 0,
+               0, 0, 1, 0, 0, 0],
+            P=[0.1, 0, 0, 0, 0, 0,
+                0, 0.1, 0, 0, 0, 0,
+                0, 0, 0.1, 0, 0, 0,
+                0, 0, 0, 1, 0, 0,
+                0, 0, 0, 0, 1, 0,
+                0, 0, 0, 0, 0, 1],
+            G=[0.5, 0.5, 0.5, 1, 1, 1],
+            R=[1, 0, 0,
+                0, 1, 0,
+                0, 0, 1],
+            # fmt: on
+        )
+    )
+
+    @classmethod
+    def get_scaled_matrix(
+        cls, name: str, sigma: float, use_cell_config: bool = True
+    ) -> List[float]:
+        """Returns the scaled version (i.e. the unscaled matrix multiplied by sigma)
+        of the matrix.
+
+        Keyword arguments:
+        name -- the matrix name (can be one of A, H, P, G, R)
+        sigma -- the factor to scale the matrix entries with
+        cell -- whether to use cell config matrices or not (default true)
+        """
+        if name == "A":
+            if use_cell_config:
+                name = "A_cell"
+            else:
+                name = "A_particle"
+        return (np.asarray(cls().unscaled_matrices[name]) * sigma).tolist()
+
+    @classmethod
+    def get_sigma(cls, name: str, scaled_matrix: npt.NDArray[np.float64]) -> float:
+        """Returns the factor sigma which is the multiplier between the given scaled
+        matrix and the unscaled matrix of the given name.
+
+        Note: Calculation relies on the top-left entry of the matrix.
+
+        Keyword arguments:
+        name -- the matrix name (can be one of A, H, P, G, R)
+        scaled_matrix -- the scaled matrix to find sigma from.
+        """
+        if name == "A":
+            name = "A_cell"  # doesn't matter which A we use here, as [0][0] is the same
+        return scaled_matrix[0][0] / cls().unscaled_matrices[name][0]
+
+
+def run_tracker(
+    segmentation: Union[napari.layers.Image, napari.layers.Labels],
+    tracker_config: TrackerConfig,
+) -> Tuple[npt.NDArray, dict, dict]:
+    """
+    Runs BayesianTracker with given segmentation and configuration.
+    """
     with btrack.BayesianTracker() as tracker:
-        # configure the tracker using a config file
-        tracker.configure_from_file(config_file_path)
+        tracker.configure(tracker_config)
         tracker.max_search_radius = 50
```

Comment:
`max_search_radius` should be exposed to the user

Reply:
Exposed.